tests/system/small/test_remote_function.py

```diff
@@ -465,6 +465,36 @@ def square(x):
     assert_pandas_df_equal_ignore_ordering(bf_result, pd_result)
 
 
+@pytest.mark.flaky(retries=2, delay=120)
+def test_remote_function_default_connection(scalars_dfs, dataset_id):
+    @rf.remote_function([int], int, dataset=dataset_id)
```

Comment:
direct version is deprecated, recommend testing `session.remote_function` or `bigframes.pandas.remote_function` interface.

https://github.com/googleapis/python-bigquery-dataframes/blob/228aeba09782ae2421040c7601c15d4af92790b6/bigframes/remote_function.py#L609

Reply:
I think it's okay, the goal of this test is to assert that default connection takes effect irrespective of the session.

We can add a separate large test `test_remote_function_direct_defaults_to_global_session` specifically for that change.